how-to-extend/adapter-starterkit/conf/includes/dataSourceCustom.conf

```diff
@@ -0,0 +1,3 @@
+# Event bus address of the example Knot
+address = ${dataSource.custom.address}
+params.message = Hello Knot.x
```

Comment:
Looks like this param is never used in options.

Reply:
Idea was to provide the basic configuration. As the implementation expects `params.message` I put it here.

Reply:
adapterRequest.getParams().getString("message")